_docs/release-notes.md

```diff
@@ -306,7 +306,19 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
 
 If you already have the Tidepool Uploader installed on your computer, it will automatically update to the latest version.  
 
-### 2.6.0 (Release 2018-06-04)
+### 2.6.2 (Released 2018-06-13)
+We fixed some of our code that deals with identifying the right time for your data. If you really want to dig into our UTC Bootstrapping algorithm, send us an email and we'll talk your ear off about the challenges of device times.  
+[Feature Requirements](https://trello.com/c/rh2oLpjR/8-apply-utc-bootstrapping-fix-for-invalid-timezone-offset-to-new-data)
+
+We're running on node-serialport v6 now. You may not think that's exciting, but now devices that connect through a serial port will automatically retry an upload if it fails the first time. And you thought dependency updates were boring.  
```

Comment:
I realise this is a bit pedantic, but the auto-retry code is something I added after updating to v6.

Reply:
How about: "You may not think that's exciting, but this update let us add code to automatically retry an upload..."

Reply:
Full edit: "We're running on node-serialport v6 now. You may not think that's exciting, but this update let us add code to automatically retry an upload the first time a device connected through a serial port fails. And you thought dependency updates were boring."

Reply:
Maybe "You may not think that's exciting, but it fixes a bunch of issues for devices that connect through a serial port on Windows. And you thought dependency updates were boring. We now also automatically retry a serial port upload if it fails the first time. "?

Reply:
I like that. I'll make the update and merge the PR.